docs/gettingstarted/manifest-deployer-tutorial/README.md

```diff
@@ -0,0 +1,358 @@
+
+1. The Target resource. This tells the Landscaper **where** it should deploy the application to. Landscaper currently supports only one target type, a kubernetes cluster.
+
+    ``` yaml
+    apiVersion: landscaper.gardener.cloud/v1alpha1
+    kind: Target
+    metadata:
+      creationTimestamp: null
+      name: my-cluster
+      namespace: example
+    spec:
+      config:
+        kubeconfig: |+
+        <KUBECONFIG>
+
+      type: landscaper.gardener.cloud/kubernetes-cluster
+    ```
+
+    Either replace the `<KUBECONFIG>` placeholder with your clusters kubeconfig, or use the landscaper-cli tool (recommended), to create the Target manifest:
+
+    ```bash
+    landscaper-cli targets create kubernetes-cluster --name my-cluster --namespace example --target-kubeconfig ~/.kube/config-demo.yaml > ./my-target.yaml
+    ```
+
+2. The Installation resource. While the Blueprint provided a specification of all Import parameters, the Installation provides concrete values for the Imports. Furthermore, the Installation makes a connection to the component-descriptor associated with the application we are going to deploy. The below Installation specifies the component-descriptor, it specifies which blueprint it shall use (a component-descriptor can contain more than one blueprint). As for the Imports, there is just one parameter which needs to be initialized. There is an Import of type target with the name target. This is the same Target resource, we created in step 1. You find a sample Installation in /manifests/Installation.yaml. Replace the `<OCIURL>` placeholder with the OCI registry URL you created in section "Install the Landscaper together with an OCI registry".
+
+    ``` yaml
+    apiVersion: landscaper.gardener.cloud/v1alpha1
+    kind: Installation
+    metadata:
+      name: manifest-demo
+      namespace: example
+    spec:
+      blueprint:
+        ref:
+          resourceName: blueprint
+
+      componentDescriptor:
+        ref:
+          componentName: test.net/test
+          repositoryContext:
+            baseUrl: <OCIURL>
+            type: ociRegistry
+          version: v0.1.0
+
+      imports:
+        targets:
+          - name: cluster
+            target: '#my-cluster'
+    ```
+
+3. Create **example** namespace
+
+    ``` bash
+    kubectl create namespace example
+    ```
+
+4. Apply the custom resources to your cluster:
+
+    ``` bash
+    kubectl apply -f ./my-target.yaml
+    kubectl apply -f ./installation.yaml
+    ```
+
+5. Verify the status of the deployment:
+
+    ``` bash
+    landscaper-cli installations inspect manifest-demo -n example
+    ```
+
+    The final status should look like this:
+
+    ``` bash
+    >$ landscaper-cli installations inspect manifest-demo -n example
+    [✅ Succeeded] Installation manifest-demo
+        └── [✅ Succeeded] DeployItem manifest-demo-default-deploy-item-ncflh
+    ```
+
```

Comment:
What about verifying the deployed manifests?

Reply:
Yep, a kubectl get deployment manifest-demo -n example is missing here